apps/io/feeding_service_webhook.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2016 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.io.commands import update_ingest
+from superdesk.io.registry import registered_webhooks
+import superdesk
+
+
+class FeedingServiceWebhookResource(Resource):
+    public_methods = ['GET', 'POST']
```

Comment:
usually it's good practice to have some kind of  `Signature` for example: https://developer.github.com/webhooks/#delivery-headers
```
X-Hub-Signature	HMAC hex digest of the payload, using the hook's secret as the key (if configured).
```
that you can identify that the posted request is part that has secret key...

I don't understand why Wufoo don't use it, this `a handshake parameter` I think is just `GET` parameter like secret, but the secret shouldn't be transferred over internet...

so in case of Wufoo probably IP restriction should be used or so.